dev/custom-pages/internet-privacy.html

```diff
@@ -0,0 +1,39 @@
+<div class="row"><div class="col-md-12">
+When you visit our Web site, our Web server automatically recognizes only the Internet domain and IP address from which you accessed our site. This process does not identify your email address or other personal information.
+
+Our Web server compiles statistics regarding the volume and timing of access to our Web site, such as the time and pages accessed. We do this so that we can improve the content and structure of our Web site, and only aggregate information is collected.
+
+If members of the general public choose to share personal information with us by sending us a message, or filling out an electronic form, we will use the information only for the purposes authorized. Some of the information may be saved for a designated period of time to comply with the state of Florida's archiving policies, but we will not disclose information about public visitors to this site with third parties or other government agencies, unless required to do so by Florida or federal law. And if we are required to disclose information by law, we will make every effort to notify any affected parties in advance.
+
+Under current Florida and Federal law, certain personal student information is made available on the main Web site except for those students who elect to have their information suppressed from this site. Such students should read the instructions on the page <a href="http://www.ucf.edu/internet-privacy-policy/privacy-of-student-information/">"Privacy of Student Information on the UCF Web Site"</a>.
+
+If you have questions about UCF's Internet Privacy Policy, please feel free to email us at <a href="mailto:web@mail.ucf.edu">web@mail.ucf.edu</a> or contact us by telephone at 407-823-5117.
+
+<h2>Digital Millennium Copyright Act Agent</h2>
+The agent designated by the University of Central Florida to receive and act on copyright violations under the Digital Millennium Copyright Act (DMCA) is:
+
+Joel L. Hartman, Vice Provost and Chief Information Officer
+Millican Hall 338K
+4000 Central Florida Blvd.
+Orlando, FL 32816
+407-823-6778 (voice)
+407-823-6710 (fax)
+Joel.Hartman@ucf.edu
+
+The University's policies with regard to information technology, copyright, and related matters are available on the World Wide Web at the following URL: <a href="http://policies.ucf.edu" target="_blank">http://policies.ucf.edu</a>
+
+Additional information on the Digital Millennium Copyright Act (DMCA) can be found at: <a href="http://www.educause.edu/library/digital-millennium-copyright-act-dmca" target="_blank">http://www.educause.edu/Resources/Browse/DMCA/31236</a>
+
+<h2>Privacy of Student Information</h2>
+The information displayed on the UCF web site is consistent with UCF policy on the release of directory information. Your information can be suppressed by following the directions provided below.
+<p><strong>Directions for Suppressing Information</strong></p>
```

Comment:
This is probably copied straight from the existing page, but this line and line 35 should probably be h3's.